.github/workflows/ci.yml

```diff
@@ -17,13 +17,15 @@ jobs:
           - "3.9"
           - "3.10"
           - "3.11"
+          - "3.12"
```

Comment:
Based on the CI results, look  like we need to exclude Django 2.2 running against Python 3.12: